verification/envs/python_3/main.py

```diff
@@ -17,29 +15,16 @@ def _make_id(target):
     return id(target)
 
 
-def run_events():
-    while not EVENTS_CALLS.empty():
-        send_event = EVENTS_CALLS.get()
-        send_event()
-
-
-class PlayerRunner(Runner):
-
-    def __init__(self):
-        super().__init__()
-        self._is_executing = False
-
-    @property
-    def is_executing(self):
-        return self._is_executing
+class PlayerRefereeRunner(Runner):
+    def __init__(self, *args, **kwargs):
+        self._events = {}
+        super().__init__(*args, **kwargs)
 
-    def execute(self, execution_data):
-        self._is_executing = True
-        return super().execute(execution_data)
+    def action_event(self, data):
```

Comment:
actually, I can't change it. 'action_' is a predefined prefix.
Same as in parent class 
action_stop
action_config
etc